### ISSUE TITLE
Training spacy(ner) with custom training data

### ISSUE THREAD

**Issue by Aditya-9712** [explosion/spaCy#1990]
Hello @honnibal,
I have collected custom training data for training(updating) spacy's NER. I am currently using the en_core_web_md English model which I am looking to update. 
How should I go about this. Help would be appreciated.

## Your Environment

* **spaCy version:** 2.0.7
* **Platform:** Windows-7-6.1.7601-SP1
* **Python version:** 3.6.4
* **Models:** en, en_core_web_md

**Comment by Aditya-9712** [comment]
This is a sample of the training data that I have collected.

**('Neither City A.M. nor Digital look to accept any responsibility', { 'entities': [(8,18,'ORG')]})**

I want to use multiple such collected data points to update the model being used.

**Comment by ahalterman** [comment]
If you have data in that format and there's isn't very much of it, you can use the NER training code in [`examples/training/train_ner.py`](https://github.com/explosion/spaCy/blob/master/examples/training/train_ner.py). More info about how it works is in the [docs](https://spacy.io/usage/training#ner).

If you have a large amount of data, it may be faster to train using the `spacy train` [CLI](https://spacy.io/api/cli#train), but I've been having trouble getting it to take just NER labels without POS, dependency, etc (see #1907).
